package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "escape-html",
-  "description": "Escape HTML entities",
+  "description": "Escape HTML special characters (syntax)",
```

Comment:
What does putting "(syntax)" add to the description? I'm also not sure many people understand what a "special character" is, so it should be a very simplistic description (but can easily be expanded in the readme).